app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -9,6 +9,7 @@ def self.ems_inv_to_hashes(inventory)
 
     def initialize
       @data = {}
+      @data[:container_image_registries] ||= []
```

Comment:
@simon3z I tested this and there is no need to initialize this in case no registries are found. 
In case no registries are found then `@data[:container_image_registries]` will return nil, and then `hashes[:container_image_registries]` in [1] will be nil and in [2] we will return without a failure.

[1]https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_refresh/save_inventory_container.rb#L13
[2]https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_refresh/save_inventory_container.rb#L325